src/org/opendatakit/briefcase/model/BriefcaseFormDefinition.java

```diff
@@ -365,6 +366,15 @@ public boolean isFileEncryptedForm() {
   }
 
   public TreeElement getSubmissionElement() {
+    TreeElement treeElement = formDefn.getSubmissionElement();
```

Comment:
I changed the call to use getSubmissionElement() instead of calling directly from submissionElement.